ema_workbench/util/utilities.py

```diff
@@ -64,6 +64,9 @@ def load_results(file_name):
             except TypeError:
                 dtype = pd.api.types.pandas_dtype(dtype)
 
+            if experiments[name].dtype is not dtype:
+                experiments[name] = experiments[name].astype(dtype)
```

Comment:
Can we add a log/print/warning statement here that notes that there was a data type conversion? That might make it possible to identify future bugs or unintended behaviour.

Something like:
```Python
f"Data type of {name} was incorrectly loaded as {experiments[name].dtype}, converting back to {dtype}."
```

Reply:
That won't really work. We rely on pandas.read_csv to load the csv. This does a best guess dtype. But if you have e.g., integers that refer to a category in a categorical uncertainty (as happens with e.g., vensim models quite often), read_csv will load this as an integer dtype which then is converted into a categorical dtype (for correct handling in e.g. PRIM this is quite relevant).

Reply:
Right, thanks for the explanation.